test/application/team-validator.js

```diff
@@ -34,6 +34,22 @@ describe('Team Validator features', function () {
 			});
 		});
 
+		it('should reject non-existent natures', function (done) {
+			let packedTeam = "|pikachu|eviolite||thunderbolt|nonexistentNature||||||";
+			TeamValidator('customgame').prepTeam(packedTeam).then(result => {
```

Comment:
So you'd rather have them replaced with something like this then? I mostly follow the same style for the tests that's already made, because this is very unfamiliar to me.
```
it('should reject non-existent natures', function (done) {
   	let team = [{species:'pikachu', moves:['thunderbolt'], nature:'nonexistentNature'}];
   	let illegal = TeamValidator('customgame').validateTeam(team);
   	if (illegal) return done();
   	return done(new Error("Non-existent nature accepted"));
});
```